.github/workflows/hotfix.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       - name: Check out the code
         uses: actions/checkout@v4
         with:
+          token: ${{ secrets.GHA_ELEVATED_PERMISSIONS_TOKEN }}
```

Comment:
Is the elevated permissions token really needed also for checkout?

Reply:
Thanks for taking time to dig deeper into it. It's not a big deal if the token is required, and we keep it around for situations like this. I'm fine with using that token here if it unblocks the workflow 👍